src/style/_misc.scss

```diff
@@ -3,6 +3,7 @@ $animation-duration: 250ms;
 $z-index-cat-nav: 1; // the top nav must be above the hero image on the overview page
 $z-index-header: 2; // so the shadow is above the panel below
 $z-index-left-nav: 3; // must be on top when it opens on mobile
+$z-index-feedback-button: 4; // floats over content
```

Comment:
You'll probably need to bump this to five when you merge master in, I've added a layer.

Reply:
tooltip gets the 5